aten/src/THC/generic/THCStorage.cpp

```diff
@@ -6,11 +6,6 @@
 #include <c10/util/typeid.h>
 
 
-scalar_t* THCStorage_(data)(THCState *state, const THCStorage *self)
-{
-  return self->data<scalar_t>();
-}
-
 int THCStorage_(elementSize)(THCState *state)
```

Comment:
I think (elementSize) can also be removed (even though it's not the goal of this PR)

Reply:
Done. There may be other functions that can be easily removed too, but if so, I'll include them in future PRs. I'm splitting up the TH/THC Storage removal into several PRs